docs/handbook/community.md

```diff
@@ -34,6 +34,100 @@ Link to PDF Document: <a href= "../pdf/HowToInstallOracleDB.pdf" target="_blank"
 <p>Unfortunately, it is not possible to run Oracle Database version 12 and above on OpenIndiana.</p>
 </div>
 
+# Hipster Handbook - Squeak Smalltalk-80
```

Comment:
Can we preserve only brief description here and move information to the separate page (to match the common style of this page)?